.github/workflows/python.yml

```diff
@@ -128,6 +128,52 @@ jobs:
               with:
                   language-flag: -python
 
+    test-pubsub-ubuntu-latest:
+      runs-on: ubuntu-latest
+      needs: load-engine-matrix
+      timeout-minutes: 35
+      strategy:
+          fail-fast: false
+          matrix:
+              engine: ${{ fromJson(needs.load-engine-matrix.outputs.matrix) }}
+              python:
+                - "3.8"
+                - "3.9"
+                - "3.10"
+                - "3.11"
+                - "3.12"
+
+      steps:
+          - uses: actions/checkout@v4
+            with:
+                submodules: recursive
+
+          - name: Set up Python
+            uses: actions/setup-python@v4
+            with:
+                python-version: ${{ matrix.python }}
+
+          - name: Install dependencies
+            working-directory: ./python
+            run: |
+                python -m pip install --upgrade pip
+                pip install mypy-protobuf
```

Comment:
You don't need this, you don't test mypy in this job

Reply:
right, removed